doc/dev/kubernetes.rst

```diff
@@ -83,7 +83,7 @@ This is likely to be the same host you're using as your kubernetes master.
 3. Build Rook
 =============
 
-.. info:
+.. info::
```

Comment:
I guess `info` doesn't exist, and you need `.. note::`  here

Reply:
thanks - updated